pkg/landscaper/controllers/deployitem/controller.go

```diff
@@ -40,30 +43,54 @@ func NewController(log logr.Logger, c client.Client, scheme *runtime.Scheme, raw
 		}
 		con.pickupTimeout = &tmp
 	}
+	if rawAbortingTimeout == "none" {
+		con.abortingTimeout = nil
+	} else {
+		tmp, err := time.ParseDuration(rawAbortingTimeout)
+		if err != nil {
+			return nil, fmt.Errorf("unable to parse deploy item aborting timeout into a duration: %w", err)
+		}
+		con.abortingTimeout = &tmp
+	}
+	if rawDefaultTimeout == "none" {
+		con.defaultTimeout = nil
+	} else {
+		tmp, err := time.ParseDuration(rawDefaultTimeout)
+		if err != nil {
+			return nil, fmt.Errorf("unable to parse deploy item default timeout into a duration: %w", err)
+		}
+		con.defaultTimeout = &tmp
+	}
```

Comment:
if we use my propsed change to the api. this code will become unnecessary and way less complex